src/FieldWarning/Assets/Ingame/UI/SpawnPointBehaviour.cs

```diff
@@ -11,55 +11,46 @@
  * the License for the specific language governing permissions and limitations under the License.
  */
 
-using UnityEngine;
 using System.Collections.Generic;
 using System.Linq;
-
-public class SpawnPointBehaviour : MonoBehaviour
-{
-    public PFW.Model.Game.Team Team;
-
+using PFW.Model.Game;
+using UnityEngine;
+public class SpawnPointBehaviour : MonoBehaviour {
     public const float MIN_SPAWN_INTERVAL = 2f;
     public const float QUEUE_DELAY = 1f;
+    /* I broke C# law by using  */
+    public Team @Team { get; private set; }
+    private Queue<GhostPlatoonBehaviour> spawnQueue { get; } = new Queue<GhostPlatoonBehaviour>();
 
-    private Vector3 oldPosition;
-    private Queue<GhostPlatoonBehaviour> spawnQueue = new Queue<GhostPlatoonBehaviour>();
-    private float spawnTime = MIN_SPAWN_INTERVAL;
+    private float _spawnTime = MIN_SPAWN_INTERVAL;
```

Comment:
Nice style cleanup but you missed the name fix for spawnQueue :+1:

Reply:
That's what I get for trusting VSC linter. :'[